### ISSUE TITLE
Accordion Panel within an Accordion Panel is broken

### ISSUE THREAD

**Issue by rosenthalj** [primefaces/primeng#11848]
### Describe the bug

A p-accordion within p-accordion  is broken in primeNG v14.0.1. 

The contents of all the p-accordionTab panels of an embedded p-accordion panel is  always visible when the  embedded p-accordion panel is visible

![AccordionWithAccordion](https://user-images.githubusercontent.com/45439491/185700645-5bb09772-6a75-4679-a6b1-df967161eae8.gif)

https://user-images.githubusercontent.com/45439491/185698133-f293ede4-4342-4bca-8fc2-6b2330a6805a.mov

I believe the bug was induced by #11617 commit 
 https://github.com/primefaces/primeng/commit/f849e9d23b0c2223b6fbb516d3b0c368c927cf22

![Fixed__11617_-_PrimeNG_component_animation_trigger_warnings_since_upd…_·_primefaces_primeng_f849e9d](https://user-images.githubusercontent.com/45439491/185705188-dce978e8-206b-4898-83d1-7d07149bd73d.gif)

### Environment

Happens in both stackblitz and local project

### Reproducer

https://stackblitz.com/edit/github-irxb1p?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.spec.ts

### Angular version

14.x

### PrimeNG version

14.0.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

14+

### Browser(s)

All Browsers

### Steps to reproduce the behavior

https://github-irxb1p.stackblitz.io

1. Expand all header tabs
2. Randomly open and close Header A and Header B tabbed panel.
- See that the contents of Header A and Header B are always visible

### Expected behavior

Embedded PrimeNG Accordion panels work the same as non-embedded PrimeNG Accordions.  

V14.x embedded p-accordion works the same as V13.x  embedded p-accordion

**Comment by majkers** [comment]
The same as https://github.com/primefaces/primeng/issues/11736?

**Comment by krtek4** [comment]
If a dropdown is bigger than the size of the accordion, its content isn't displayed entirely.

Instead of just removing the buggy css, something like that could be done :

```
.p-accordion-tab-active > .p-toggleable-content:not(.ng-animating) {
  overflow: visible;
}
```

**Comment by kAgotsi** [comment]
Hello, I'm facing same issue, I tried fix :

**Comment by ger1995** [comment]
The issue with the nested Accordions seemed to be fixed in 4.1.0 but appears to occur again in 4.1.1/4.1.2, see https://stackblitz.com/edit/angular-ivy-q8x2la?file=package.json
